pkg/redshift/api/api.go

```diff
@@ -381,7 +384,7 @@ func (c *API) Clusters() ([]models.RedshiftCluster, error) {
 }
 
 func (c *API) Workgroups() ([]models.RedshiftWorkgroup, error) {
-	out, err := c.ServerlessManagementClient.ListWorkgroups(&redshiftserverless.ListWorkgroupsInput{})
+	out, err := c.ServerlessManagementClient.ListWorkgroups(context.TODO(), &redshiftserverless.ListWorkgroupsInput{})
```

Comment:
Would it be easy to update the function to accept a context in this PR? The three places in this file that use `context.TODO()` looks like it could have it done with only a few changes.